pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -146,6 +156,14 @@ func (s *volumeStatCalculator) parsePodVolumeStats(podName string, pvcRef *stats
 	}
 }
 
+func isVolumeEphemeral(volume v1.Volume) bool {
+	if (volume.EmptyDir != nil && volume.EmptyDir.Medium == v1.StorageMediumDefault) ||
+		volume.ConfigMap != nil || volume.GitRepo != nil {
```

Comment:
What about downward API? Also need to check for projected volumes with a configmap source.

Reply:
both downwardAPI and projected volume use strorageMemory medium, so it does not use disk.